linkerd/app/src/env.rs

```diff
@@ -599,7 +599,7 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
             let mut ports = match parse(strings, ENV_INBOUND_PORTS, parse_port_range_set)? {
                 Some(ports) => ports.into_iter().flatten().collect::<HashSet<_>>(),
                 None => {
-                    error!("No inbound ports specified via {}", ENV_INBOUND_PORTS,);
+                    debug!("No inbound ports specified via {}", ENV_INBOUND_PORTS,);
```

Comment:
```suggestion
                    debug!("No inbound ports specified via {ENV_INBOUND_PORTS}");
```

While you're here, you could modernize the format syntax.